CMakeLists.txt

```diff
@@ -483,14 +485,12 @@ install (
 #==============================================================================
 
 # create and install pkgconfig file
-if (UNIX OR MINGW)
-  configure_file (nlopt.pc.in nlopt.pc @ONLY)
+configure_file (nlopt.pc.in nlopt.pc @ONLY)
```

Comment:
you can imagine compiling the library using MSVC, and later using it from MinGW with pkg-config.
Hence I installed the .pc file, same thing with the man pages..